.github/workflows/submitty_ci.yml

```diff
@@ -520,3 +520,14 @@ jobs:
       - uses: ./.github/actions/e2e-Setup-Composite
       - name: Run integration tests
         run: sudo -E env "PATH=$PATH" python3 /usr/local/submitty/test_suite/integrationTests/run.py
+
+  Notify-Failure:
+    runs-on: ubuntu-latest
+    if: ${{ always() && contains(join(needs.*.result, ','), 'failure') && github.event.pull_request.title }}
+    needs: [css-lint, js-lint, js-unit, php-lint, php-unit, python-lint, python-unit, yaml-lint, ShellCheck, Integration, Cypress-System, Cypress]
```

Comment:
Is there a way to say `needs: [*]` or similar so you don't have to specify all of the different jobs?  I didn't see anything like that in my brief look through the docs, but it would be a good change if it is possible to do.

Reply:
I don't see one, if I try needs[*] the github actions syntax doesn't allow it.